ledger/src/shred.rs

```diff
@@ -334,6 +334,16 @@ macro_rules! dispatch {
                 Self::ShredData(shred) => shred.$name($($arg, )?),
             }
         }
+    };
+    (allow_dead $vis:vis fn $name:ident(&self $(, $arg:ident : $ty:ty)?) $(-> $out:ty)?) => {
```

Comment:
temporarily adding this for clippy checks, will remove in #102

Reply:
Instead of adding a new variant for the macro, can you just add `#[allow(dead_code)]` in line 313?